.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+# Overview
+
+Addresses ticket [#TICKET-NUMBER](www.ticket-link.com)
+
+High level overview of changes; one or two sentences.
+
+The design for the ticket is based on:
+![This design from XXXXXX](screenshot-of-design.png)
+
+# Changes
+
+## Major
+
+- Major change one
+- Major change two
+- ...
+
+## Minor
+
+- Minor change one
+- Minor change two
+- Minor change three
+- ...
+
```

Comment:
Suggest a "How to review" section, so if its a bug thats tricky to reproduce or something it can offer some guidance

Reply:
Added something but not quite happy with it, comments welcome.